### DISCUSSION TITLE
Should we don't show YouTube shorts?

### DISCUSSION THREAD

**Discussion by arthur-fontaine** [wanned/karmine-corp-app#193] [Polls]
<img width="385" alt="Capture d’écran 2024-06-12 à 23 07 07" src="https://github.com/wanned/karmine-corp-app/assets/57626613/45956b2d-6645-40de-b578-f9d1a2d66116">

**Comment by iPaulux** [comment]
I think we should #KCWIN

**Reply by arthur-fontaine** [reply]
Mdrrrr comment t'es tombé dessus
